node/src/command.rs

```diff
@@ -549,27 +539,22 @@ pub fn run() -> Result<()> {
 				info!("Parachain genesis state: {}", genesis_state);
 				info!("Is collating: {}", if config.role.is_authority() { "yes" } else { "no" });
 
-				match &config.chain_spec {
-					#[cfg(feature = "mangata-kusama")]
-					spec if spec.is_mangata_kusama() => crate::service::start_parachain_node::<
-						service::mangata_kusama_runtime::RuntimeApi,
-						service::MangataKusamaRuntimeExecutor,
-					>(config, polkadot_config, collator_options, id, hwbench)
-					.await
-					.map(|r| r.0)
-					.map_err(Into::into),
-					#[cfg(feature = "mangata-rococo")]
-					spec if spec.is_mangata_rococo() => crate::service::start_parachain_node::<
-						service::mangata_rococo_runtime::RuntimeApi,
-						service::MangataRococoRuntimeExecutor,
-					>(config, polkadot_config, collator_options, id, hwbench)
+				if !collator_options.relay_chain_rpc_urls.is_empty() && cli.relay_chain_args.len() > 0 {
+					warn!("Detected relay chain node arguments together with --relay-chain-rpc-url. This command starts a minimal Polkadot node that only uses a network-related subset of all relay chain CLI options.");
+				}
+
+				crate::service::start_parachain_node(
+					config,
+					polkadot_config,
+					collator_options,
+					id,
+					hwbench,
+				)
 					.await
 					.map(|r| r.0)
-					.map_err(Into::into),
-					_ => panic!("invalid chain spec"),
-				}
+					.map_err(Into::into)
 			})
-		},
+		}
```

Comment:
that should be reverted, or aligned with what we have on develop (runtime & executor explicitly passed)